### DISCUSSION TITLE
How to resolve 404 error when the src of the iframe does not contain .html under ifame preview mode?

### DISCUSSION THREAD

**Discussion by wangqunmu** [web-infra-dev/rspress#1268] [Q&A]
尝试过设置route.cleanUrls，但依然不管用
![image](https://github.com/user-attachments/assets/d1cef25a-f13f-413d-9234-976d922a25cc)

**Comment by Timeless0911** [comment]
提一个 issue，并且提供下复现的仓库和复现的方式~
